sample/F8/speakers

```diff
@@ -0,0 +1,2512 @@
+[{
+		"id": 1,
+		"name": "Mike Arcuri",
+		"email": null,
+		"mobile": null,
+		"photo": null,
+		"organisation": "Facebook",
+		"country": "Not available",
+		"short_biography": "",
+		"long_biography": "",
+		"website": null,
+		"twitter": null,
+		"facebook": null,
+		"github": null,
+		"linkedin": null,
+		"featured": false,
+		"position": null,
+		"sessions": []
+
+	},
+	{
+		"id": 2,
+		"name": "Michael Slater",
+		"email": null,
+		"mobile": null,
+		"photo": null,
+		"organisation": "Facebook",
+		"sessions": [{
```

Comment:
"sessions" array is at last in speaker object. It would be good to follow same pattern as generated by server.